arch/riscv/Kconfig

```diff
@@ -671,6 +671,18 @@ config MMU_SV32
 
 endmenu # "Platform type"
 
+config XUANTIE_ISA
+	bool "XUANTIE ISA in AFLAGS with -march=_xtheadc"
+	default n
+	depends on $(cc-option, -march=rv64imafdcv_xtheadc)
```

Comment:
Why not rv64imafdc_xtheadc

Reply:
There is no "march=rv64imafdc_xtheadc" build option. You can enable KBUILD_VERBOSE = 1 to check it.